packages/blast-utilities/keypair.js

```diff
@@ -3,8 +3,11 @@ const bip32 = require('bip32')
 const secp256k1 = require('secp256k1')
```

Comment:
not directly connected to adding accounts but keypair.js can be renamed to something more accurate such as "accounts.js" or "account-utils.js"